### ISSUE TITLE
Error when using TabViewItem as a stop on Android

### ISSUE THREAD

**Issue by lrilling** [hamdiwanis/nativescript-app-tour#23]
I recently encountered a problem when trying to include a stop for a `TabViewItem`. 
It gave me the error 
```
Error: java.lang.IllegalArgumentException: Given null view to target
``` 
I found out that `refToVueView.nativeView.android` is not defined, nor is `refToVueView.android`. But `refToVueView.nativeView.nativeView`, actually returns the actual android nativeView for every kind of UI Element not only `TabViewItems` but also `Buttons` and `Labels`, which is also others confirmed to me in the {N} discord. 

I had to change the `buildNativeTour` method in a way to call TapTarget as following: 
```return TapTarget.forView(stop.view.nativeView, stop.title, stop.description|| this.defaults.description)```

(before it was
```return TapTarget.forView(stop.view.android, stop.title, stop.description|| this.defaults.description)```
)
So that might already resolve the issue. 

But I am not sure if this only happens for `nativescript-vue`, I will try to reproduce it on a plain `nativescript` project and could open a PR for the change once I found out if the problems exists for plain nativescript aswell.

**Comment by lrilling** [comment]
Still have some problem with using `<ActionItem>` as a stop because I can't find a way to access it's nativeView, also the way described above does not work for these kind of items, but that problem is probably connected to `nativescript-vue` will try to find a fix to that aswell, but if someone has an idea...

**Comment by lrilling** [comment]
Ok to access an `ActionItem` I actually had to make kind of a detour: 
```
this.$refs.searchbutton.nativeView.parent.nativeView.getChildAt(0)
```
But this already returns a nativeView, which is why I had to change the `app-tour.android.ts` to 
```
return TapTarget.forView(stop.view.nativeView || stop.view, stop.title, stop.description|| this.defaults.description)
```
doesn't break how it worked before.

**Comment by hamdiwanis** [comment]
Thanks, @lrilling for your time investigating the issue and I see you have good progress already. will be happy to accept your PR. 😃

**Comment by lrilling** [comment]
I included my changes in the PR #24

**Comment by hamdiwanis** [comment]
Fixed by PR #24
